content/contributing/sprint-guide/contents.lr

```diff
@@ -0,0 +1,209 @@
+What does that mean?
+
+GitHub provides a list of issues that allows BeeWare (and other projects) to
+keep track of all the problems that have been reported - that includes explicit
+bugs, unexpected behavior that *might* be a bug, requests for new APIs, requests
+for more documentation - anything that might be a way that BeeWare could be
+improved. As new issues are reported, the Bee Team will flag some of them as
+issues that are well suited to a first time contributor - issues that don't
+demand an *immediate* fix, but are well defined or easy to reproduce, and
+probably have a good lead on the underlying cause. These issues are tagged "good
+first issue" to make them easier to find.
+
+To filter a GitHub issue list by the "good first issue" label, follow these steps:
+
+1. Click on the "Labels" button above the list of issues to activate the drop-down menu.
+2. Type the word "good" into the search box at the top of the drop-down menu.
+3. Click on "good first issue" in the filtered list.
+
+.. figure:: ./github-good-first-issue-filter.png
+    :width: 100%
+    :alt: The GitHub issue label filter drop-down menu activated with a search for the word "good" to filter down to "good first issue".
+
+    *Filtering GitHub issues by the "good first issue" label.*
+
+Once you've found an issue that looks interesting, read through any comments on
+the post, and post a comment in the discussion saying that you're looking into
+the issue (e.g., "Looking into this at the PyCon US 2025 sprints"). BeeWare
+doesn't explicitly "assign" issues - we accept contributions from anyone.
+However, in a sprint situation, we need to make sure that we don't end up with
+two people working on the same issue - leaving a comment is a light-weight way
+to coordinate between contributors.
+
+If you still can't find anything you'd like to do, find a Bee Team member and
+they'll help you narrow down to a specific issue.
+
+Set up a development environment
+---------------------------------
+
+Once you've found an issue, you'll need to set up your development environment
+so you can work on a fix. Each BeeWare project has it's own contribution guide:
+
+* `Briefcase <https://briefcase.readthedocs.io/en/latest/how-to/contribute/index.html>`__
+* `Toga <https://toga.readthedocs.io/en/latest/how-to/contribute/index.html>`__
+* `Rubicon ObjC <https://rubicon-objc.readthedocs.io/en/latest/how-to/contribute/index.html>`__
+
+If the project isn't on this list, check the project README for details on how
+to get set up. If any of the instructions don't make sense (or you can't find
+those instructions), find a member of the Bee Team, and they'll get you unstuck.
+And - if you think you can improve the instrutions so other people won't hit the
+same problem, improving the contribution guide is a valuable contribution!
+
+Now you can start working on your issue!
+
+What happens when I'm done?
+----------------------------
```

Comment:
```suggestion
What happens after I submit my PR?
----------------------------------
```
I feel like "when I'm done" doesn't necessarily harmonise with the content below it, as a huge part of your point here is that basically everyone will go through a change request. In the case of new folks, I think it's important to be as clear as possible about each step of the process.